### PR TITLE
Add organization Tagion

### PR DESCRIPTION
**Pull request by imrying**

### CODE REVIEW

File: orgs-using-d.dd [hunk 1]
```diff
@@ -222,6 +222,15 @@ $(DIVC orgs-using-d center,
             $(FA_HIRING github.com/symmetryinvestments/overview/blob/master/hiring.md)
         )
     )
+    $(DORG Tagion, https://www.tagion.org/, tagion.org,
+        Decentralized DLT network,
```
Comment (thewilsonator) [inline orgs-using-d.dd:226]: You might want to clarify that DLT means (I'm assuming) Distributed ledger technology here. Not everyone is going to know what that means. 

(Also the linked site does not say what DLT is/means, it assumes you know).

Comment (imrying) [reply orgs-using-d.dd:226]: Thanks... On it ;-)

Comment (imrying) [reply orgs-using-d.dd]: Is this better @thewilsonator?

**Comment by thewilsonator** [comment]
cc @mdparker

**Review by mdparker — APPROVED** [review]

**Pull request merged**